MyNUnit/MyNUnit/Utils/TestResult.cs

```diff
@@ -0,0 +1,5 @@
+namespace MyNUnit;
+
+public record TestResult(string TestName, bool IsPassed, Type? ExceptionType, string? ExceptionMessage, string? IgnoreMessage , TimeSpan Time);
+
+public record TestResultClass( string ClassName, List<TestResult> TestResults);
```

Comment:
```suggestion
public record TestResultClass(string ClassName, List<TestResult> TestResults);
```
Стоит вынести его в отдельный файл